keyboards/cool836A/cool836A.h

```diff
@@ -0,0 +1,52 @@
+/* Copyright 2021 Ohashi
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "quantum.h"
+
+/* This is a shortcut to help you visually see your layout.
+ *
+ * The first section contains all of the arguments representing the physical
+ * layout of the board and position of the keys.
+ *
+ * The second converts the arguments into a two-dimensional array which
+ * represents the switch matrix.
+ */
+
+/*#define LAYOUT( \
+    k00, k01, k02, \
+      k10,  k12    \
+) { \
+    { k00, k01,   k02 }, \
+    { k10, KC_NO, k12 }  \
+}*/
+#define LAYOUT( \
+  K000, K001, K002, K003, K004, K005, \
+  K100, K101, K102, K103, K104, K105, \
+  K200, K201, K202, K203, K204, K205, \
+  K300, K301, K302, K303, K304, K305, \
+  K400, K401, K402, K403, K404, K405, \
+  K500, K501, K502, K503, K504, K505 \
```

Comment:
This looks to be an alice like board, that is 3x12, not 6x6. 
 You can (should) change the layout macro here to reflect the actual, physical layout of the boards.

```suggestion
  K000, K001, K002, K003, K004, K005,   K300, K301, K302, K303, K304, K305, \
  K100, K101, K102, K103, K104, K105,   K400, K401, K402, K403, K404, K405, \
  K200, K201, K202, K203, K204, K205,   K500, K501, K502, K503, K504, K505 \
```

Reply:
Thank you for your suggestion.

I'd like to reply [here](https://github.com/qmk/qmk_firmware/pull/11467#discussion_r553700832).